src/shared/error/person-already-exists.error.ts

```diff
@@ -0,0 +1,7 @@
+import { DomainError } from './domain.error.js';
+
+export class PersonAlreadyExistsError extends DomainError {
```

Comment:
We are currently using the error only in the person module, but I would argue that we keep domain errors in the shared folder, because we can reuse them in any module and for other use cases.

Reply:
Leaving the decison to ticket EW-557.